docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,184 @@
 <titleabbrev>Encrypting communications</titleabbrev>
 ++++
 
-{kib} supports Transport Layer Security (TLS/SSL) encryption for all forms of
-data-in-transit. Browsers send traffic to {kib} and {kib} sends traffic to {es}.
-These communications are configured separately.
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption for data-in-transit. While these terms are often used
+interchangeably, {kib} only supports TLS, which has superseded the old SSL protocols.
+
+Browsers send traffic to {kib} and {kib} sends traffic to {es}. These communications are configured separately to use TLS.
+
+TLS requires X.509 certificates to perform encryption and authentication of the application that is being communicated with. {kib} supports
+certificates and private keys in PEM or PKCS#12 format.
 
 [[configuring-tls-browser-kib]]
 ==== Encrypting traffic between the browser and {kib}
 
-NOTE: You do not need to enable {security-features} for this type of encryption. 
```

Comment:
This note seemed unnecessary, so I removed it.

Reply:
Re-reading everything, I guess this note still makes sense because it's linked from the ["Configuring security in Kibana" page](http://kibana_55739.docs-preview.app.elstc.co/guide/en/kibana/master/using-kibana-with-security.html), which is focused on X-Pack security. I'll add it back in.